### ISSUE TITLE
Question on CIFAR-10 results

### ISSUE THREAD

**Issue by pietz** [raghavchalapathy/oc-nn_old#4]
I'm a little confused by the results published in your paper vs. the experiments you ran at KDD18. In the paper it seems that the OC-NN achieves essentially a perfect score on the unsupervised task of learning what a dog in CIFAR-10 looks like and telling it apart from a cat in CIFAR-10.

This is truly remarkable on it's own because current state-of-the-art CNNs that are a lot more complex than the proposed architecture don't reach this level of accuracy even if they have been trained on much larger datasets, much larger images and making it a binary classification which is the much easier task to solve.

In the Jupyter notebook from KDD18 the results are entirely different than the results reported in the paper and actually making the oc-nn the worst performing method tested.

Could you share your thoughts on this? Am I misunderstanding something?

Thanks!

**Comment by raghavchalapathy** [comment]
The data used in the experiments is shared [here](https://drive.google.com/drive/folders/1Dfymf7OMizy2ll3qGZ-uHsOASiMkfZWm)
If you observe the decision score plot, all the anomalies obtain negatives scores , and normal data are shown to obtain positive scores, if you can use np.sign( scores) and input this pred_labels to AUROC of sklearn you will observe the values reported in the paper 
This is how I have computed the pred-labels and reported the results in paper published in arxiv
I will update this in the notebook as well

PS: It is observed that for certain cases or images of ( cats, dogs ) CIFAR-10 OC-NN may not produce such good results however the dataset which is best performing is reported in the paper
  
Raghav
